modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-parameters_{context}"]
+= Ingress controller configuration parameters
+
+The `ingresscontrollers.operator.openshift.io` resource offers the following
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`domain`
+|`domain` is a DNS name serviced by the Ingress controller and is used to configure multiple features:
+
+* For the `LoadBalancerService` endpoint publishing strategy, domain is used to configure DNS records. See `endpointPublishingStrategy`.
+
+* When using a generated default certificate, the certificate will be valid for domain and its subdomains. See `defaultCertificate`.
+
+* The value is published to individual Route statuses so that end-users know where to target external DNS records.
+
+`domain` must be unique among all Ingress controllers, and cannot be updated.
+
+If empty, defaults to `ingress.config.openshift.io/cluster` `.spec.domain`.
+
+This parameter is optional.
+
+|`replicas`
+|`replicas` is the desired number of Ingress controller replicas. If not set, defaults to 2.
+
+|`endpointPublishingStrategy`
+|`endpointPublishingStrategy` is used to publish the Ingress controller endpoints to other networks, enable load balancer integrations, and so on.
```

Comment:
> ! and so on n
Use only when you list a clear sequence of elements, such as “1, 2, 3, and so on”

maybe s/and so on/and provide access to other systems